src/Psl/Filesystem/change_group.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Filesystem;
+
+use Psl;
+
+/**
+ * Change the group ownership of $filename.
+ *
+ * @param bool          $recursive  Whether change the group ownership recursively or not.
+ *
+ * @throws Exception\RuntimeException   If unable to change the group ownership for $filename.
+ * @throws Psl\Exception\InvariantViolationException If $filename does not exist.
+ */
+function change_group(string $filename, int $group, bool $recursive = false): void
```

Comment:
with function names, i decided to go with something more descriptive than what php ( also c, c++, unix .. ) has, but i think having more descriptive names are better, and are more newbie friendly ( e.g: `change_permissions` vs `chmod`, `create_file` vs `touch`, `create_directory` vs `mkdir`, `delete_file` instead of `unlink`, `delete_directory` instead of `rmdir` )

Reply:
change_group
change_group_recursive